lib/taskana-core/src/test/java/acceptance/task/QueryTaskCommentAccTest.java

```diff
@@ -0,0 +1,253 @@
+package acceptance.task;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import acceptance.AbstractAccTest;
+import java.time.Instant;
+import java.util.List;
+import org.apache.ibatis.exceptions.TooManyResultsException;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import pro.taskana.common.api.TimeInterval;
+import pro.taskana.common.test.security.JaasExtension;
+import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.task.api.TaskCommentQuery;
+import pro.taskana.task.api.TaskCommentQueryColumnName;
+import pro.taskana.task.api.models.TaskComment;
+
+/** Test for TaskComment queries. */
+@ExtendWith(JaasExtension.class)
+public class QueryTaskCommentAccTest extends AbstractAccTest {
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_ReturnHistoryEvents_For_DifferentInAttributes() {
+    List<TaskComment> comments =
+        taskService
+            .createTaskCommentQuery()
+            .idIn(
+                "TCI:000000000000000000000000000000000000",
+                "TCI:000000000000000000000000000000000002")
+            .list();
+    assertThat(comments).hasSize(2);
+
+    comments =
+        taskService
+            .createTaskCommentQuery()
+            .taskIdIn("TKI:000000000000000000000000000000000000")
+            .list();
+    assertThat(comments).hasSize(3);
+
+    comments = taskService.createTaskCommentQuery().creatorIn("user-1-2").list();
+    assertThat(comments).hasSize(2);
+
+    TimeInterval timeInterval = new TimeInterval(Instant.now().minusSeconds(10), Instant.now());
+    comments = taskService.createTaskCommentQuery().createdWithin(timeInterval).list();
+    assertThat(comments).isEmpty();
+
+    comments = taskService.createTaskCommentQuery().modifiedWithin(timeInterval).list();
+    assertThat(comments).isEmpty();
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_ReturnHistoryEvents_For_DifferentNotInAttributes() {
```

Comment:
historyEvents? i think this is a copy pasta fail. This also applies for all the other tests in this class